audio/pcm_decode.c

```diff
@@ -151,8 +151,10 @@ static uint16_t pcm_leuint32(uint32_t value);
 #  define pcm_leuint32(v) (v)
 #endif
 
+#ifndef CONFIG_AUDIO_FORMAT_RAW
 static inline bool pcm_validwav(FAR const struct wav_header_s *wav);
```

Comment:
add length argument

Reply:
and now we have a second patch that fixes a potential bug :)

Reply:
Thanks for the suggestions.